### ISSUE TITLE
[Bug]: Tool Bench turns into repair bench when broken

### ISSUE THREAD

**Issue by minigamerking** [iamkaf/mod-issues#13]
### Which mod are you filling this issue for?

Bonded

### What happened?

I was upgrading my pickaxe and then broke the tool bench and it dropped a repair bench, the video is of me upgrading my diamond axe to showcase the bug

https://github.com/user-attachments/assets/0d9e7816-4e51-4e58-8c79-25797213c3c3

### Minecraft Version

1.21.4

### Which mod loader are you using?

Fabric

### Mod Loader Version

16.5

### Mod List

```shell
Accelerated Decay
Amber
Anvil Restoration
AppleSkin
Attack Through Grass
Better Statistics Screen
BetterF3
BetterThanMending
Bonded
CList
Controlling
Dark Loading Screen
Detail Armor Bar
Do a Barrel Roll
Dungeons and Taverns
Entity Model Features
Entity Texture Features
EntityCulling
EXPCounter
Explorify
Fast Minecart
FerriteCore
Fish On The Line
Geophilic
GUI Compass
Iris
Jade
Lithium
Mod Menu
Mouse Tweaks
Polymer
Pottery
Quick Right-Click
Real Arrow Tip
Refined Advancements
Resource Backpacks
Resourcify
Revamped Shipwrecks
REI
Scribble
Server Translations API
Sit
Sodium
SuperMartijn642's Config Lib
SuperMartijn642's Core Lib
Tax Free Levels
Toggleable Item Frames
Trimmable Tools
Universal Graves
Unloaded Activity
Zoomify
```

**Comment by iamkaf** [comment]
LOL OMG. A silly mistake on my part, I'll fix it right away! Thank you for reporting.

**Comment by iamkaf** [comment]
I fixed it on `1.1.1`.
